docs/application/dotnet/guides/nui/resources.md

```diff
@@ -61,26 +90,26 @@ Visual creation automatically determines whether the image is N_PATCH, SVG, or G
 | `CacheSize`   | Number of images to keep cached, can increase or decrease depending on memory available. |
 | `FrameDelay`  | Millisecond delay between frames. |
 
-Synchronous loading has niche uses, for example, ensure image has loaded before continuing to execute application code,
+Synchronous loading has niche uses, for example, ensure image has loded before continuing to exceute application code,
```

Comment:
Spell error. Plse update as below:

Synchronous loading has niche uses, for example, ensure image has loaded before continuing to execute application code